libraries/chain/abi_serializer.cpp

```diff
@@ -400,15 +400,15 @@ namespace eosio { namespace chain {
       auto h = ctx.enter_scope();
       auto rtype = resolve_type(type);
       auto ftype = fundamental_type(rtype);
-      auto btype = built_in_types.find(ftype );
-      if( btype != built_in_types.end() ) {
+      bool var_is_array = is_szarray(rtype);
```

Comment:
I think this can be:
`bool var_is_array = is_array(rtype) || is_szarray(rtype);`
and then just `var_is_array` below.

Reply:
I see your point, but maybe we want a different serialization for fixed-size arrays without the size, as Matt's mentioned in telegram. Let le try that.